ddtrace/internal/runtime/lightstep_metrics.py

```diff
@@ -0,0 +1,215 @@
+import logging
+import os
+import platform
+import random
+import string
+
+from ... import _worker
+
+from .constants import GC_RUNTIME_METRICS
+from .metric_collectors import GCRuntimeMetricCollector, RuntimeMetricCollector
+from .runtime_metrics import RuntimeCollectorsIterable
+from google.protobuf.duration_pb2 import Duration
+from google.protobuf.timestamp_pb2 import Timestamp
+from ddtrace.vendor.lightstep.collector_pb2 import KeyValue, Reporter
+from ddtrace.vendor.lightstep.metrics_pb2 import IngestRequest, MetricKind
+
+log = logging.getLogger(__name__)
+
+
+LS_PROCESS_CPU_TIME_SYS = "process.cpu.system"
+LS_PROCESS_CPU_TIME_USER = "process.cpu.user"
+LS_PROCESS_MEM_RSS = "process.mem.rss"
```

Comment:
should these three be prefixed with `runtime.python.*`?

Reply:
Fixed.